Assets/Scripts/GamepadController.cs

```diff
@@ -147,8 +147,20 @@ private void FixedUpdate()
 
         // Gamepad has been assigned
         slotSelected = true;
+
+		// Kick any other gamepads out of this slot
+		transform.parent.BroadcastMessage("leaveOccupiedSlot", slot);

```

Comment:
Woah cool, so any object with this method will call it when you broadcast the message? So all other GamepadControllers call it?

Reply:
That's correct... in general, any children will receive BroadcastMessage, which is why I called it on transform.parent... so all the GameControllers will call this function. Actually perfect for this use case!